modules/EvseSecurity/doc.rst

```diff
@@ -1,22 +1,69 @@
 
-..  This file is a placeholder for an optional single file handwritten documentation for 
-    the EvseSecurity module.
-    Please decide weather you want to use this single file,
-    or a set of files in the doc/ directory.
-    In the latter case, you can delete this file.
-    In the former case, you can delete the doc/ directory.
-    
-..  This handwritten documentation is optional. In case
-    you do not want to write it, you can delete this file
-    and the doc/ directory.
-
-..  The documentation can be written in reStructuredText,
-    and will be converted to HTML and PDF by Sphinx.
-
-*******************************************
+====================================
 EvseSecurity
-*******************************************
+====================================
 
-:ref:`Link <everest_modules_EvseSecurity>` to the module's reference.
-This module implements the evse_security interface. It uses the filesystem to store certificates and keys
+This module implements the `evse_security <https://github.com/EVerest/everest-core/blob/main/interfaces/evse_security.yaml>`_ interface. It wraps the functionality of the `libevse-security <https://github.com/EVerest/libevse-security>`_ to provide access to security-related operations to other Everest modules such as OCPP and ISO15118. These modules require similar security-related operations and shared access to certificates and keys, which this module facilitates. For detailed information about the provided functionality, please refer to the README within the `libevse-security <https://github.com/EVerest/libevse-security>`_.
+
+CA Certificate Domains
+=======================
+
+The combination of ISO15118 and OCPP defines several CA certificate domains for charging stations, addressed and covered by this module:
+
+* V2G root: Trust anchor for ISO15118 TLS communication between the charging station and the electric vehicle.
+* CSMS root: Trust anchor for TLS communication between the charging station and OCPP CSMS.
+* MF root: Trust anchor of the manufacturer to verify firmware updates.
+* MO root: Trust anchor of the Mobility Operator domain to verify contract certificates.
+
+Module Configuration
+=====================
+
+The following instructions describe how to configure the module parameters mainly for two domains: OCPP communication and ISO15118 communication.
+
+A lot of configuration parameters specify directory or file paths from which certificates and private keys are loaded or new certificates and keys are written to. Please make sure that these paths are writeable for EVerest. 
+
+Configuration for OCPP
+----------------------
+
+In OCPP and OCPP security profiles, the security level of the connection is specified as follows:
+
+* SecurityProfile 0: Unsecure transport without Basic Authentication
+* SecurityProfile 1: Unsecure transport with Basic Authentication
+* SecurityProfile 2: TLS with Basic authentication
+* SecurityProfile 3: TLS with client-side certificates
```

Comment:
- either "unsecured" (past tense, as used in the OCA white paper for OCPP 1.6J and in OCPP 2.0.1) or "insecure" (adjective)
- inconsistent capitalization